packages/cbioportal-ts-api-client/src/generated/CBioPortalAPI-docs.json

```diff
@@ -4577,6 +4577,9 @@
         "clinicalAttributeId": {
           "type": "string"
         },
+        "patientAttribute": {
```

Comment:
@dippindots do you know where this came from?

Reply:
This is introduced in @BasLee's `Binning of custom data` pull request: https://github.com/cBioPortal/cbioportal/pull/10039. This property is being used to determine if clinical data is sample level or patient level.